plans/feat-2574-google-tasks-uncomplete.md

```
@@ -0,0 +1,80 @@
+# feat(google): `tasksUncomplete` (#2574)
+
+## Background
+
+#2572 gave `tasksUpdate` `title` / `notes` / `due` and deliberately withheld
```

markdownlint-cli2 (0.23.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)